### PR TITLE
feat(aws): add support options overview

### PR DESCRIPTION
**Pull request by rpocase**
This adds an overview for how to get support on AWS, including what Pro provides, AWS frontline support, and getting expanded support from Canonical

### CODE REVIEW

File: .custom_wordlist.txt [hunk 1]
```diff
@@ -240,11 +242,13 @@ runtimes
 rustc
 rustup
 RSA
+SaaS
 scalability
 SDK
 SeaBIOS
 SEV
 SHA
+signup
```
Comment (rpocase) [inline .custom_wordlist.txt:251]: is there a better way to do this? the flag seems to be on a link which i wouldn't expect to be spell checked.

File: aws/aws-reference/pro.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
+Ubuntu Pro on AWS
```
[the remark this answers is not in the document]
Comment (rpocase) [reply aws/aws-reference/pro.rst:1]: good catch on actually linking the top level page. i tend to think a briefer than the whole marketing pitch is worth while even if there is duplication. many users won't take the chance to look through the linked content unless there is enough of a blurb to get them more interested

File: aws/aws-reference/pro.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+~~~~~~~~~~
+Ubuntu Pro offers compliance options for users that need to apply 
+CIS (level 1 and 2) or FIPS hardening. FIPS is currently supported
+on Ubuntu 18.04 and 20.04. 22.04 is currently available through the
+`fips-preview` and `fips-updates` repositories provided through the 
+FIPS pro entitlement, but these modules are not yet certified.
+
+Ubuntu Landscape
+~~~~~~~~~~~~~~~~
+All Ubuntu Pro customers are entitled to usage of
```
Comment (k-dimple) [inline aws/aws-reference/pro.rst:34]:
```suggestion
All Ubuntu Pro customers are entitled to the usage of 
```

File: aws/aws-reference/pro.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+FIPS pro entitlement, but these modules are not yet certified.
+
+Ubuntu Landscape
+~~~~~~~~~~~~~~~~
+All Ubuntu Pro customers are entitled to usage of 
+`Landscape <https://ubuntu.com/landscape>_` self-hosted and SaaS solutions.
+Landscape is an endpoint management solution for monitoring your Ubuntu
+estate. This includes alerting for security vulnerabilities, patching status,
+managing repository mirrors, and more. To sign up for Landscape SaaS see
+the `sign up page <https://landscape.canonical.com/signup>_`.
```
Comment (k-dimple) [inline aws/aws-reference/pro.rst:39]:
```suggestion
the `sign up page`_.

.. _`Ubuntu Pro`: https://ubuntu.com/aws/pro
.. _`ESM`: https://ubuntu.com/security/esm
.. _`Livepatch`: https://ubuntu.com/security/livepatch
.. _`Landscape`: https://ubuntu.com/landscape
.. _`sign up page`: https://landscape.canonical.com/signup
```

File: aws/aws-reference/support.rst [hunk 5]
```diff
@@ -0,0 +1,24 @@
+Support options on AWS
+============================
+
+Ubuntu Pro on AWS
+-----------------
+Ubuntu Pro provides numerous support options. Please see the `Pro <pro>`_ page for
```
Comment (k-dimple) [inline aws/aws-reference/support.rst:6]:
```suggestion
Ubuntu Pro provides numerous support options. Please see the :doc:`Pro <pro>` page for 
```

File: aws/aws-reference/support.rst [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+Ubuntu Pro on AWS
+-----------------
+Ubuntu Pro provides numerous support options. Please see the `Pro <pro>`_ page for 
+more details.
+
+
+Amazon front line support
+-------------------------
+Canonical and Amazon collaborate on publishing Ubuntu Pro images
+through the `Amazon Web Services marketplace seller <https://aws.amazon.com/marketplace/search/results?searchTerms=ubuntu+pro&CREATOR=e6a5002c-6dd0-4d1e-8196-0a1d1857229b&filters=CREATOR>`_.
```
Comment (k-dimple) [inline aws/aws-reference/support.rst:13]:
```suggestion
through the `Amazon Web Services marketplace <https://aws.amazon.com/marketplace/search/results?searchTerms=ubuntu+pro&CREATOR=e6a5002c-6dd0-4d1e-8196-0a1d1857229b&filters=CREATOR>`_.
```

**Review by k-dimple — CHANGES_REQUESTED** [review]

**Comment by k-dimple** [comment]
Also, could you pull the latest code from the main repo, so that the build failures on RTD can be avoided? I have pushed a small update to remove that error.

**Comment by rpocase** [comment]
batch committed all of dimple's suggestions. i'll follow up tomorrow on carlos' additional comments and review the custom_wordlist to see what I can remove from before

**Review by k-dimple — APPROVED** [review]

**Pull request merged**